great_expectations/expectations/window.py

```diff
@@ -23,6 +23,7 @@ class Window(pydantic.BaseModel):
     parameter_name: str
     range: int
     offset: Offset
+    strict: bool
```

Comment:
@shiplet is adding a new required parameter here a breaking change? I don't see windows anywhere in the docs or gallery, which tells me it isn't really part of the public API. I think we are probably fine to make this required, but I could add a default if there is risk.